src/ScottPlot/Enums/AxisChart.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ScottPlot
+{
+    public enum AxisChart
```

Comment:
I would prefer if this was agnostic to what's rendering the `StarAxis` and instead just said where to put the images.

e.g.:
```cs
enum ImagePlacement {
    Outside,
    Inside
}
```

Reply:
This avoids hiding what the field changes and it is more extensible in case we wanted to create a new plottable which used `StarAxis`, or if we wanted the image placement to be customizable.

Reply:
So, some values like Arms or the area between the arms, how should it be called?

Your points are fair and sound, thanks!

Reply:
Makes sense, will do it this way!